src/main/java/com/collegecompendium/backend/models/User.java

```diff
@@ -72,32 +74,8 @@ public static HashMap<String, String> getAuth0(Jwt token) {
 		if (response.getStatusCode().is2xxSuccessful()) return response.getBody();
 		return null;
 	}
-	
-	public String getEmail() {
-		return email;
-	}
-	public void setEmail(String email) {
-		this.email = email;
-	}
-	public String getLocation() {
-		return location;
-	}
-	public void setLocation(String location) {
-		this.location = location;
-	}
-	public String getId() {
-		return id;
-	}
-	
+		
 	public void setId(String val) {
 		this.id = val;
```

Comment:
Is this still needed?

Reply:
We have it set id to Null in new student creation